ztag/schema.py

```diff
@@ -35,20 +35,22 @@ class CensysString(WhitespaceAnalyzedString):
     "value":IndexedBinary(),
 })
 
+edi_party_name = SubRecord({
+    "name_assigner":CensysString(),
+    "party_name":CensysString(),
+})
+
 alternate_name = SubRecord({
     "dns_names":ListOf(FQDN()),
     "email_addresses":ListOf(EmailAddress()),
     "ip_addresses":ListOf(IPAddress()),
     "directory_names":ListOf(zgrab_subj_issuer),
-    "edi_party_names":ListOf(SubRecord({
-        "name_assigner":CensysString,
-        "party_name":CensysString,
-    })),
+    "edi_party_names":ListOf(edi_party_name),
     "other_names":ListOf(SubRecord({
-        "id":String(),
+        "id":String(), # XXX is this an OID?
```

Comment:
Yes, this is an OID. https://github.com/zmap/zgrab/blob/ad824319f6286ff5fbfe85b4605a672d8c13ba15/ztools/x509/pkix/json.go#L171